IsTheWay5_1.py

```diff
@@ -0,0 +1,7 @@
+from os import listdir
+
+def beginWithDeep(path):
```

Comment:
1. Please add documantation to the function.
2. Nice job using the list comprehension and the "startswith" method.